### PR TITLE
fix/metamask popup

### LINKED ISSUE #78: perf: metamask request to sign into the app at the loading
### Acceptance criteria

- [ ] Metamask Pop up should appear only when the user click on SIWE button.

### PR DESCRIPTION
**Pull request by dasarathg68**
# Description

Fixes #78 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

* **Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines

### CODE REVIEW

**Comment by hermannleboss** [comment]
@dasarathg68 i added an acceptance criteria on the issue. Take a look

File: app/src/components/NavBar.vue
[the remark this answers is not in the document]
Comment (dasarathg68) [reply app/src/components/NavBar.vue:1]: image
image

**Comment by farrellh1** [comment]
i think i've fixed it in #75 on App.vue file, u can review it

**Comment by dasarathg68** [comment]
> i think i've fixed it in #75 on App.vue file, u can review it

Yes, I think it's better to close this one

File: app/src/components/NavBar.vue
[the remark this answers is not in the document]
Comment (dasarathg68) [reply app/src/components/NavBar.vue:1]: Yes, but this is resolved in #75 by Farrell.

**Pull request closed without merging**